### PR TITLE
Week 3 - Step 4 - Using Forms and POST 

### PR DESCRIPTION
**Pull request by malcolmrjones**

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -1,74 +1,85 @@
...
+        <a href="https://www.linkedin.com/in/malcolmrjones" target="_blank">LinkedIn</a>
+        <a href="https://www.github.com/malcolmrjones" target="_blank">Github</a>
+        
+        <hr>
+
+        <h2>My Projects</h2>
+        <ul>
+          <li>cipes</h3>
+            <h4>REU - University of North Carolina at Charlotte</h4>
+            <p>Web app that provides recipe recommendations based off of ingredient preferences. Techk Stack: HTML, CSS, JavaScript, Django Web Framework</p>
```
Comment (brucezheng) [inline portfolio/src/main/webapp/index.html]: Nit: Tech Stack

**Review by brucezheng — APPROVED** [review]

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -1,74 +1,85 @@
...
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+        <h1>Malcolm Jones Portfolio</h1>
+
+        <img src="/images/malcolm-1.jpeg" width="500" height="500">
+
+        <h2>About Me</h2>
+        <p>
+          Welcome to my portfolio website! I am Malcolm and I am from Winston-Slame, NC. 
+          Currently I am studying computer science at North Carolina Agricultural and Technical State Universty.
```
Comment (ccyang314) [inline portfolio/src/main/webapp/index.html:21]: nit: typo `University`

Comment (ccyang314) [inline portfolio/src/main/webapp/index.html]: nit: typo `University`

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -12,17 +12,40 @@
...
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addGreeting() {
+  const greetingPromise = fetch("/data");
+  greetingPromise.then(handleGreeting);
+}
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+function handleGreeting(response) {
```
Comment (ccyang314) [inline portfolio/src/main/webapp/script.js:29]: nit: add JS function comment to all the new functions included in this PR. 

https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments

**Review by ccyang314 — CHANGES_REQUESTED** [review]

**Review by ccyang314 — APPROVED** [review]
LGTM. Thanks! 

For newer PRs, try to merge to previous step's branch instead of master branch to avoid noisy diffs.

**Pull request merged**